OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs

```diff
@@ -62,7 +62,7 @@ public void Run(ModData modData, string[] args)
 					{
 						var n = require.Name;
 						var name = n.EndsWith("Info") ? n.Remove(n.Length - 4, 4) : n;
-						doc.Append("`{0}`{1}".F(name, i + 1 == reqCount ? ".\n" : ", "));
+						doc.Append("[{0}](#{1}){2}".F(name, name.ToLowerInvariant(), i + 1 == reqCount ? ".\n" : ", "));
```

Comment:
You can still keep the formatting with `doc.Append("[`{0}`](#{1}){2}".F(name, name.ToLowerInvariant(), i + 1 == reqCount ? ".\n" : ", "));`